core/binding.rbs

```diff
@@ -30,9 +30,9 @@
 # Binding objects have no class-specific methods.
 #
 class Binding
-  def clone: () -> self
+  def clone: () -> instance
```

Comment:
Do you have any specific reason to use `instance` for those methods?
I think using `self` is enough and better here. It always returns a `Binding` object.

Reply:
I was thinking `self` means "the exact same object" (i.e. with `def foo: () -> self`, `self.foo.equal?(self)` wouldalways be true), whereas `instance` means an instance of `Binding`.

Reply:
`self` is the exactly same type of the receiver.
It's different from `instance` if the type is generic. `instance` type fills the generic parameters by `untyped`, so `Array[String]` will be `Array[untyped]`.

They are identical because `Binding` is not generic, but I think `self` is more preferable.